Hi @andrm 

Thanks for your contribution!

`qt5-base` fails to build on Linux with the following message:

```
CMake Error at ports/qt5-base/install_qt.cmake:33 (file):
  file COPY cannot copy file "/usr/bin/flex" to "/usr/bin/bin/flex".
Call Stack (most recent call first):
  ports/qt5-base/portfile.cmake:147 (install_qt)
  scripts/ports.cmake:71 (include)


Error: Building package qt5-base:x64-linux failed with: BUILD_FAILED
```

You should wrap the relevant code inside an if block like this

```
if (CMAKE_HOST_WIN32)
   # your code
endif()
```

--- a/ports/qt5-base/CONTROL
+++ b/ports/qt5-base/CONTROL
@@ -1,4 +1,4 @@
 Source: qt5-base
-Version: 5.12.3
+Version: 5.12.3-1
 Description: Qt5 Application Framework Base Module. Includes Core, GUI, Widgets, Networking, SQL, Concurrent and other essential qt components.
 Build-Depends: zlib, libjpeg-turbo, libpng, freetype, pcre2, harfbuzz, sqlite3, libpq, double-conversion, openssl

--- a/ports/qt5-base/install_qt.cmake
+++ b/ports/qt5-base/install_qt.cmake
@@ -25,22 +25,21 @@ function(install_qt)
     endif()
     vcpkg_find_acquire_program(PYTHON3)
     get_filename_component(PYTHON3_EXE_PATH ${PYTHON3} DIRECTORY)
-    vcpkg_find_acquire_program(FLEX)
-    get_filename_component(FLEX_EXE_PATH ${FLEX} DIRECTORY)
-    #    vcpkg_find_acquire_program(BISON)
-    #get_filename_component(BISON_EXE_PATH ${BISON} DIRECTORY)
-
-    file(COPY ${FLEX} DESTINATION "${FLEX_EXE_PATH}/bin" )
-    file(COPY "${FLEX_EXE_PATH}/win_bison.exe" DESTINATION "${FLEX_EXE_PATH}/bin" )
-    file(RENAME "${FLEX_EXE_PATH}/bin/win_bison.exe" "${FLEX_EXE_PATH}/bin/bison.exe")
-    file(RENAME "${FLEX_EXE_PATH}/bin/win_flex.exe" "${FLEX_EXE_PATH}/bin/flex.exe")
-    file(COPY "${FLEX_EXE_PATH}/bin/flex.exe" DESTINATION "${FLEX_EXE_PATH}" )
-    file(COPY "${FLEX_EXE_PATH}/bin/bison.exe" DESTINATION "${FLEX_EXE_PATH}" )
-    vcpkg_add_to_path("${FLEX_EXE_PATH}")
-
     vcpkg_add_to_path(PREPEND "${PYTHON3_EXE_PATH}")
 
-    set(_path "$ENV{PATH}")
+    # flex and bison for ANGLE library
+    vcpkg_find_acquire_program(FLEX)
+    get_filename_component(FLEX_EXE_PATH ${FLEX} DIRECTORY)
+    get_filename_component(FLEX_DIR ${FLEX_EXE_PATH} NAME)
+
+    file(COPY ${FLEX_EXE_PATH} DESTINATION "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-tools" )
+    set(FLEX_TEMP "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-tools/${FLEX_DIR}")
+    message(STATUS "FLEX_TEMP: ${FLEX_TEMP}")
+    file(RENAME "${FLEX_TEMP}/win_bison.exe" "${FLEX_TEMP}/bison.exe")
+    file(RENAME "${FLEX_TEMP}/win_flex.exe" "${FLEX_TEMP}/flex.exe")
+    vcpkg_add_to_path("${FLEX_TEMP}")
+
+   set(_path "$ENV{PATH}")
 
     if(NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")
         message(STATUS "Package ${TARGET_TRIPLET}-dbg")

--- a/ports/qt5-base/install_qt.cmake
+++ b/ports/qt5-base/install_qt.cmake
@@ -40,10 +40,6 @@ function(install_qt)
 
     vcpkg_add_to_path(PREPEND "${PYTHON3_EXE_PATH}")
 
-
-    message(STATUS "PATH: $ENV{PATH}")
-
-
     set(_path "$ENV{PATH}")
 
     if(NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")

